communis-mission/src/main/java/com/rhema/communis/mission/domain/Event.java

```diff
@@ -1,45 +1,23 @@
 package com.rhema.communis.mission.domain;
 
+import lombok.Getter;
+import lombok.Setter;
+
 import java.time.LocalDate;
 import java.time.LocalTime;
 
+@Getter
+@Setter
 public class Event extends Ministry {
 
     private EventType type;
     private LocalDate localDate;
     private LocalTime startTime;
     private LocalTime endTime;
 
-    public LocalDate getLocalDate() {
-        return localDate;
-    }
-
-    public void setLocalDate(LocalDate localDate) {
-        this.localDate = localDate;
-    }
-
-    public LocalTime getStartTime() {
-        return startTime;
-    }
-
-    public void setStartTime(LocalTime startTime) {
-        this.startTime = startTime;
-    }
-
-    public LocalTime getEndTime() {
-        return endTime;
-    }
-
-    public void setEndTime(LocalTime endTime) {
-        this.endTime = endTime;
-    }
-
     @Override
     public MinistryType getType() {
         return MinistryType.EVENT;
```

Comment:
@anilaselwyn  I have never used lombok before. However, I am just wondering how does it behave when you want it to use your own getter method defined? Is that why you haven't removed it?

Reply:
From the documentation - https://projectlombok.org/features/GetterSetter, looks like lombok will skip generating that function, if it sees that there is already a function that exists with that name:

```
No method is generated if any method already exists with the same name (case insensitive) and same parameter count. For example, getFoo() will not be generated if there's already a method getFoo(String... x) even though it is technically possible to make the method. This caveat exists to prevent confusion. If the generation of a method is skipped for this reason, a warning is emitted instead. Varargs count as 0 to N parameters. You can mark any method with @lombok.experimental.Tolerate to hide them from lombok.
```

Reply:
Yes that's right @jmathewt 
Also this article explains it well - https://dzone.com/articles/project-Lombok-boilerplate-code-reducer
Checkout class EmployeeWithoutLLombok Vs EmployeeWithLombok.
The annotations autogenerates the getters and setters.
You may not be want to use it if you want a override a usual getter and setter like the above override you pointed out